NitroxServer/Serialization/ServerConfig.cs

```diff
@@ -166,13 +50,50 @@ public string GenerateRandomString(int size, bool lowerCase)
 
         public void ChangeServerAdminPassword(string pw)
         {
-            _serverAdminPassword = pw;
+            serverAdminPasswordSetting.Value = pw;
 
             Configuration config = ConfigurationManager.OpenExeConfiguration(ConfigurationUserLevel.None);
-            config.AppSettings.Settings[DEFAULT_PASSWORD_SETTING].Value = pw;
+            config.AppSettings.Settings[serverAdminPasswordSetting.Name].Value = pw;
             config.Save(ConfigurationSaveMode.Modified);
 
             ConfigurationManager.RefreshSection("appSettings");
         }
+    }
+    
+    internal class ServerConfigItem<T>

```

Comment:
This needs to be in its own file.

Reply:
It was actually nested in ServerConfig till i pull it out. Do we have too extract every helper class? I dont have much experience on teamwork or huge project but as i can see, too much files of internal helper class will just mess devs minds up.

Reply:
Nah, most devs use the search facilities within the IDE to find the code they are looking for. I think the default hotkey for VS is pressing sift twice, then you can just type the name of the class you want. Trying to solve the problem of solution clutter in any other way never scales well. 

I don't think that will be too much of a problem here though. It looks like those classes are in their small namespace anyways.

Reply:
ok